components/automate-gateway/handler/iam/v2/introspect/introspect.go

```diff
@@ -29,18 +29,18 @@ import (
 
 // AuthzServer is the server interface
 type AuthzServer struct {
-	client        authz.AuthorizationClient // now only needed for GetVersion
-	filterHandler middleware.SwitchingFilterHandler
+	client               authz.AuthorizationClient // now only needed for GetVersion
+	introspectionHandler middleware.IntrospectionHandler
 }
 
 // NewServer creates a server with its client
 func NewServer(
 	client authz.AuthorizationClient,
-	filterHandler middleware.SwitchingFilterHandler,
+	IntrospectionHandler middleware.IntrospectionHandler,
```

Comment:
Does this need to be capitalized?

Reply:
Nope, good catch. I am not good at all at search-and-replace. :D